### ISSUE TITLE
Models missing after some days

### ISSUE THREAD

**Issue by tuetenk0pp** [nextcloud/recognize#1036]
### Which version of recognize are you using?

5.0.3

### Enabled Modes

Face recognition

### TensorFlow mode

WASM mode

### Downstream App

Memories App

### Which Nextcloud version do you have installed?

27.1.3

### Which Operating system do you have installed?

debian 12

### Which database are you running Nextcloud on?

postgres

### Which Docker container are you using to run Nextcloud? (if applicable)

all-in-one

### How much RAM does your server have?

8GB

### What processor Architecture does your CPU have?

x86_64

### Describe the Bug

from here: [nextcloud/all-in-one/#3791](https://github.com/nextcloud/all-in-one/discussions/3791)

> I installed the recognize app algong with the memories app in my aio and ran `occ recognize:download-models` to download the models. But after a few days I noticed that the recognize app was no longer working and I had to run the command again. I assume that aio recreated the container in the meantime.

### Expected Behavior

I should not have to redownload the models.

### To Reproduce

use the recognize app with AIO

**Comment by szaimen** [comment]
@marcelklehr is it possible that the models are stored in a location which is not persistant maybe? This could explain why they are missing. Apart from thar I have no clue why they should disappear...

**Comment by marcelklehr** [comment]
Yes, after updating the app the models are gone. Sadly there is currently no better way. We cannot store them in app data, for example. After discussing this in the team, we decided leave this as is, and add a setup check that warn admins. Long term we'll move the model part to an exapp anyway.
